specutils/manipulation/extract_spectral_region.py

```diff
@@ -161,3 +161,36 @@ def extract_region(spectrum, region):
         extracted_spectrum = extracted_spectrum[0]
 
     return extracted_spectrum
+
+
+def spectral_slab(spectrum, s1, s2):
```

Comment:
```suggestion
def spectral_slab(spectrum, lower, upper):
```

For consistency with other bounds-related functionality in specutils (e.g. `SpectralRegion`), we should reference the range using similar verbiage.